src/components/books/list.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] As before, this could be a react component. Please, convert the component into a react one. 🐱

Reply:
Changed for a class based component